### ISSUE TITLE
When table names match built-in CFML functions, the generated code will not run

### ISSUE THREAD

**Issue by seancorfield** [tpryan/Apptacular#11]
I have a table called valueList and Apptacular generated the following erroneous code:

```
Invalid CFML construct found on line 49 at column 50.

ColdFusion was looking at the following text:
valuelist

The CFML compiler was processing:

A script statement beginning with remote on line 49, column 9.

The error occurred in /Applications/ColdFusion9/wwwroot/ws/services/valuelistService.cfc: line     49
47 :    * @hint Updates one record from valuelist.
48 :    */
49 :    remote void function update(required any valuelist ) {
50 :        arguments.valuelist.nullifyZeroID();
51 :        EntitySave(arguments.valuelist);
```

**Comment by seancorfield** [comment]
A solution might to be add a prefix or suffix on argument names (perhaps $ or _ since those don't appear in built-in function names).

**Comment by seancorfield** [comment]
This also affects the generated bean names:

```
import ws.cfc.*;

<cfset valueList = new valueList() />
```

The generated bean names also need some sort of prefix/suffix as well as all references to the entity name.

**Comment by tpryan** [comment]
Hmmm.  Interesting problem. will have to dwell on that.  I think a suffix of _ is in order.

**Comment by tpryan** [comment]
Closed by 3f515f520655e02151d32b595d1479e98949fe52
http://github.com/tpryan/Apptacular/issues#issue/11
